ts/src/bitget.ts

```diff
@@ -1691,7 +1846,7 @@ export default class bitget extends Exchange {
             amountString = Precise.stringSub (amountString, feeCostAbsString);
         }
         return {
-            'id': this.safeString (transaction, 'id'),
+            'id': this.safeString2 (transaction, 'id', 'data'),
```

Comment:
as `withdraw` response (which has only `data` field, set to transaction id) is passing through `parseTransaction` , we need `data` prop too.